### ISSUE TITLE
where to buy 32port version?

### ISSUE THREAD

**Issue by kuradi** [SuperHouse/I2CRJ45#1]
hello,
have been trying to email, asking question in youtube but i still dont have answer where can i buy 32 port version? it will be on stock at all? when? 

gunnar

**Comment by austinscreations** [comment]
jon's been busy building the new lab and working on larger client work.

it'll likely take time before more are available. 

if / when they are they are available here. https://www.superhouse.tv/product/i2c-rj45-light-switch-breakout/

**Comment by kuradi** [comment]
thank you @austinscreations , finally some answer. (y)
maybe you can answer... those 16 ports... are those possible to extend, i mean can i tie 2 of them to behave as 32 ports?

**Comment by austinscreations** [comment]
Yes, you can tie two 16 port together. But you should know that two 16 ports will sit in a longer tpace than a 32 port. So you wouldn't be easily able to swap them around later. 

If you look at the mechanical pdfs you'll see what I mean about sizing and spacing.

**Comment by jonoxer** [comment]
Thanks Austin for following this up. I appreciate it!

I know that stock of the I2C breakouts has been a problem for a long time. After being prompted by this request (and previous requests on the livestreams and in Discord) I pulled out the box of parts and PCBs for them today and found the stencil for the 32-port version after moving it to the factory.

I showed my son the boards and explained what needs to be done to populate them. He's assembled probably nearly 1000 boards for me over the last couple of years and he's really good at it, so I'm going to have him work through the different sizes and assemble enough to get them all into healthy stock levels. I have 100 of the MCP23017 ready to go, which is enough to populate a few.

As Austin said, new stock will show up on the SuperHouse site.

**Comment by kuradi** [comment]
thank you, wonderful news! thank you!
as i am totally newbie of superhouse boards and switches... is it possible integrate Light Switch RJ45 Breakout (4-Button) (https://www.superhouse.tv/product/light-switch-rj45-breakout-4-button/) with shelly wall switches (https://www.shelly.cloud/en-ee/products/product-overview/shelly-wall-switch-4-white/shelly-wall-switch-4-black)?
